playbook/app/pb_kits/playbook/pb_kit/dateTime.ts

```diff
@@ -1,90 +1,173 @@
+const ABBR_DAYS = ['SU', 'M', 'T', 'W', 'TH', 'F', 'S']
 
-import moment, { Moment } from 'moment'
-import 'moment-strftime'
-import 'moment-timezone'
+const days = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"];
 
-type DateTimeType = {
-  value: string | Date,
-  zone?: string,
-}
+const months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"]
 
-const ABBR_DAYS = ['SU', 'M', 'T', 'W', 'TH', 'F', 'S']
-
-export default class DateTime {
-  value: Moment & any
-  constructor({ value, zone = 'America/New_York' }: DateTimeType) {
-    this.value = this.convertToTimestampZone(value, zone)
-  }
+const formatDate = (newDate: Date | string) => {
+  const isTimelessStringDate = typeof newDate === "string" && !newDate.includes("T")
 
-  convertToTimestampZone(value: string | Date, zone: string) {
-    return moment(value).tz(zone)
+  if (isTimelessStringDate) {
+    const unhyphenatedDate = new Date(newDate.replace(/-/g, "/"))
+    return unhyphenatedDate
   }
 
-  convertToTimezone() {
-    return this.value.strftime('%Z')
-  }
+  return new Date(newDate)
+}
```

Comment:
Format the date only if it's a `string` AND does not contain a time. This is necessary because it could be in `ISO` format already (ex. `"2012-08-02T17:49:29Z"` as shown in the [Time Range Inline](https://playbook.powerapp.cloud/kits/time_range_inline/react) example).